src/basic-syntax/compound-types.md

```diff
@@ -24,3 +24,39 @@ fn main() {
     println!("2nd index: {}", t.1);
 }
 ```
+
+<details>
+    
+Key points:
+    
+Arrays:
+    
+*Arrays have elements of the same type, T, and length, N, which is fixed. 
+
+*We can use literals to assign values to arrays.
+
+*Demonstrate out of bounds errors by setting n to different values 
+
+```
+a[n+15] = 11 // index out of bounds error since len is 10
```

Comment:
Hi @rastringer, what is `n` in this line of code?

Reply:
Hi Martin, 
I warped the video and slides in my head and missed that the 'n' isn't in the latter. Apologies and good catch. In the video, you show the out of bounds by doing:

let n = 7;
a[n+20] = 30; 
println!("a: {:?}", a);

Perhaps it's best to keep the notes simple for now and remove this part? I could update further if there's some clarity from the Internals Forum later. Nice work checking the Assembly!

Reply:
> Perhaps it's best to keep the notes simple for now and remove this part?

Yes, I'm a big fan of simplicity, so let's take it out :smile:  That should also solve the compilation error: all code blocks are checked by `mdbook test`, so you either have to make it compile (or add the `compile_fail` attribute if it's broken on purpose).

Reply:
Excellent, thanks Martin! That's good to know about the code blocks.